services/__init__.py

```diff
@@ -0,0 +1,4 @@
+import requests
+
+
+session = requests.Session()
```

Comment:
Oooh, I'd not thought about creating a global[-ish] session before, that's clever!

Reply:
Without it, you don't get the benefits of connection pooling

Reply:
Sure, I meant using a single Session instance rather than say a particular instance for slack requests.